### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by TheHeiki** [THEREALWWEFAN231/Inertia#1133]
**Inertia Utility Mod version: 3.1.3** 
**Minecraft version: 1.16.4** 
**Operating system: Windows 10** 
**Optifine/Other mods: book dupe** 

**What's the issue: When nether portal is selected in blockesp and hitbox is enabled in portal module the esp wont show up only the tracers** 

**Crash-report(if applicable)(Would be very useful also please format):**

**Final checklist:**

- [✅] I know how to properly use check boxes 
- [✅] I have tried to reproduce this bug after removing all other mods except for Inertia. 
- [✅] I have attemped to reproduce this bug using the latest available release of Inertia.

**Comment by feliscatusmeows** [comment]
I think this was fixed in a newer prebuild/next update.

**Comment by THEREALWWEFAN231** [comment]
> I think this was fixed in a newer prebuild/next update.

No, *not yet*, currently not planned, but we shall see, shouldn't' be hard, but it's not high priority.
